pkg/ocm/resources.go

```diff
@@ -0,0 +1,43 @@
+package ocm
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/openshift/assisted-service/pkg/commonutils"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/pkg/errors"
+)
+
+//go:generate mockgen -source=resources.go -package=ocm -destination=mock_resources.go
```

Comment:
this is more explicit and will generate only this interface. 
```suggestion
//go:generate mockgen -package ocm -destination mock_resources.go . OCMResources
```

Reply:
Thanks, done.